scripts-available/CDB_Overviews.sql

```diff
@@ -204,7 +231,7 @@ AS $$
         -- This is the typical ERROR: stats for "mytable" do not exist
         WHEN internal_error THEN
           -- Get stats and execute again
-          EXECUTE format('ANALYZE %1$I', reloid);
+          EXECUTE format('ANALYZE %1$s', reloid);
```

Comment:
:dancer: now I remember: when using the quick extend function, there's little chance of error if it doesn't have the stats. unlikely to happen.